### DISCUSSION TITLE
Add support for devcontainer?

### DISCUSSION THREAD

**Discussion by BrunnerLivio** [renovatebot/renovate#12114] [Request Help] [answered]
Inside of my devcontainer.json file I have a docker image referenced

```json
{
   "image": "node:14"
}
```

How do I update that?

**Comment by BrunnerLivio** [comment, marked as answer]
Add the following configuration inside your configuration

```js
{
  // Support for VSCode devcontainer.json Docker images
  regexManagers: [
    {
      fileMatch: ["\\.?devcontainer.json$"],
      matchStrings: ["\"image\" *: *\"(?<depName>.+?):(?<currentValue>.+?)\",?"],
      datasourceTemplate: "docker"
    }
  ]
}
```

**Comment by rarkins** [comment]
I think `devcontainer.json` is widely enough used now that we should consider giving it its own "manager" in Renovate

**Reply by HonkingGoose** [reply]
I've opened a issue to track/discuss this feature idea. 😉 
See:

- https://github.com/renovatebot/renovate/issues/12116
